jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/JsonExceptionMapper.java

```diff
@@ -68,7 +68,8 @@ public final Response toResponse(T exception) {
             } else {
                 String errorId = UUID.randomUUID().toString();
                 logMessage = String.format("Error %s: %s", errorId, exceptionMessage);
-                error = SerializableError.of(errorId, exception.getClass());
+                error = SerializableError.of("Refer to the server logs with this errorId: "
+                        + errorId, exception.getClass());
```

Comment:
we should probably also modify SerializableError to have explicit access to this id, but that can be for another time.